### PR TITLE
Scoping bids, contracts, and offer_contract obj

### PR DESCRIPTION
**Pull request by ljmcgann**
Uses oslo_context to scope queries to the bids,
contracts, and offer_contract relationship objects. Updated and added
new tests to accomodate these changes. TG-163

### CODE REVIEW

File: flocx_market/api/offer_contract_relationship.py [hunk 1]
```diff
@@ -9,9 +9,11 @@ class OfferContractRelationship(Resource):
...
         if (contract_id is None) and (marketplace_offer_id is None):
             return [x.to_dict() for x in
-                    ocr.OfferContractRelationship.get_all()]
-        ocrs = ocr.OfferContractRelationship.get(marketplace_offer_id,
-                                                 contract_id)
+                    ocr.OfferContractRelationship.get_all(g.context)]
+        ocrs = ocr.OfferContractRelationship.get(
+            marketplace_offer_id=marketplace_offer_id,
+            contract_id=contract_id,
+            context=g.context)
```
Comment (tzumainn) [inline flocx_market/api/offer_contract_relationship.py]: The context should not be optional. I'd put it as the first argument, like:

```
        ocrs = ocr.OfferContractRelationship.get(g.context,
            marketplace_offer_id=marketplace_offer_id,
            contract_id=contract_id)
```

Comment (ljmcgann) [reply flocx_market/api/offer_contract_relationship.py]: The context is not optional? I was just passing in the arguments out of order, as some of the calls to the offercontractrelationsip object were out of order. But in the method signature it is not optional. Or am I misunderstanding something

Comment (ljmcgann) [reply flocx_market/api/offer_contract_relationship.py]: the signature in the api is
def offer_contract_relationship_get(context,
                                    marketplace_offer_id=None,
                                    contract_id=None):

File: flocx_market/objects/offer_contract_relationship.py [hunk 2]
```diff
@@ -16,38 +16,45 @@ class OfferContractRelationship(base.FLOCXMarketObject):
     }
 
     @classmethod
-    def get(cls, marketplace_offer_id, contract_id):
+    def get(cls, marketplace_offer_id, contract_id, context):
```
Comment (tzumainn) [inline flocx_market/objects/offer_contract_relationship.py]: I'd put `context` as the first argument for consistency.

**Pull request merged**